### PR TITLE
GRW-1176 / General start date tooltip

### PR DESCRIPTION
**Pull request by amandaflod**
## What?
Add an info icon and tooltip with info about start date in the sidebar component. Initial plan was to make it car-specific but after some discussion we decided to make it more general as the problem can occur for other insurances as well!

## Why?

**Ticket(s): [GRW-1176]**

https://user-images.githubusercontent.com/59652803/172324048-a76e9ddb-ed0b-4243-956b-97e1815d314b.mov

[GRW-1176]: https://hedvig.atlassian.net/browse/GRW-1176?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/client/pages/OfferNew/index.tsx [hunk 1]
```diff
@@ -153,6 +153,8 @@ export const OfferNew = () => {
     return <LoadingPage />
   }
 
+  console.log('hellodfg fegbefgb')
```
Comment (dendoesit) [inline src/client/pages/OfferNew/index.tsx:156]: this should be removed.

**Review by dendoesit — CHANGES_REQUESTED** [review]

File: src/client/components/icons/Info.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ export const InfoIconFilled = (props: IconRootProps) => (
     <path
       fillRule="evenodd"
       clipRule="evenodd"
-      d="M14 7.00002C14 10.866 10.866 14 7.00002 14C3.13402 14 0 10.866 0 7.00002C0 3.13402 3.13402 0 7.00002 0C10.866 0 14 3.13402 14 7.00002ZM6.37503 11V6.00003H7.62503V11H6.37503ZM7.00003 4.50003C7.55232 4.50003 8.00003 4.05232 8.00003 3.50003C8.00003 2.94775 7.55232 2.50003 7.00003 2.50003C6.44775 2.50003 6.00003 2.94775 6.00003 3.50003C6.00003 4.05232 6.44775 4.50003 7.00003 4.50003Z"
+      d="M15 7.99999C15 11.866 11.866 15 8.00002 15C4.13402 15 1 11.866 1 7.99999C1 4.13399 4.13402 0.999969 8.00002 0.999969C11.866 0.999969 15 4.13399 15 7.99999ZM7.37503 12V7H8.62503V12H7.37503ZM8.00003 5.5C8.55232 5.5 9.00003 5.05228 9.00003 4.5C9.00003 3.94772 8.55232 3.5 8.00003 3.5C7.44775 3.5 7.00003 3.94772 7.00003 4.5C7.00003 5.05228 7.44775 5.5 8.00003 5.5Z"
```
Comment (simonauner) [inline src/client/components/icons/Info.tsx:21]: What is this change?

**Review by simonauner — COMMENTED** [review]
Nice one. I like that you extended the TooltipIcon with just a prop and then shift icons 👍 

I just had a question regarding the types of the component 👇

File: src/client/components/Tooltip/TooltipIcon.tsx [hunk 3]
```diff
@@ -31,10 +32,16 @@ export const TooltipIcon = (props: TooltipIconProps) => {
         onMouseLeave={() => setIsVisible(false)}
         onTouchStart={() => setIsVisible(true)}
       >
-        <InfoIcon
-          size={ICON_SIZE}
-          color={isVisible ? colorsV3.gray700 : colorsV3.gray900}
-        />
+        {filled ? (
+          <InfoIconFilled
```
Comment (guilhermespopolin) [inline src/client/components/Tooltip/TooltipIcon.tsx:36]: Shouldn't `size={ICON_SIZE}` be provided for `<InfoIconFilled />` as well? 🤔

**Review by guilhermespopolin — COMMENTED** [review]
Noice

Just add a comment :)

File: src/client/components/icons/Info.tsx (hunk 2 above)
Comment (amandaflod) [reply src/client/components/icons/Info.tsx:21]: The previous icon had a weird built in margin :/ But I have checked that it looks good in the other place it is used now as well!

File: src/client/pages/OfferNew/index.tsx (hunk 1 above)
Comment (amandaflod) [reply src/client/pages/OfferNew/index.tsx:156]: Woopsie yes sir!

File: src/client/components/Tooltip/TooltipIcon.tsx [hunk 4]
```diff
@@ -14,10 +14,11 @@ const TooltipIconWrapper = styled.div`
...
   align-items: center;
 `
+export type TooltipIconProps = Pick<TooltipProps, 'body' | 'placement'> & {
+  filled?: boolean
+}
 
-export type TooltipIconProps = Pick<TooltipProps, 'body' | 'placement'>
-
-export const TooltipIcon = (props: TooltipIconProps) => {
+export const TooltipIcon = (props: TooltipIconProps, filled = 'false') => {
```
[the remark this answers is not in the document]
Comment (amandaflod) [reply src/client/components/Tooltip/TooltipIcon.tsx:21]: You are right! Thank you :))

File: src/client/components/Tooltip/TooltipIcon.tsx (hunk 3 above)
Comment (amandaflod) [reply src/client/components/Tooltip/TooltipIcon.tsx:36]: Good catch! :D Thank you!

File: src/client/api/quoteCartQuerySelectors.ts [hunk 5]
```diff
@@ -85,3 +86,9 @@ export const isCarInsuranceType = (
     (quote) => quote.data.type === 'SWEDISH_CAR',
   )
 }
+
+export const hasCurrentInsurer = (offerData: OfferData | undefined) => {
```
[the remark this answers is not in the document]
Comment (amandaflod) [reply src/client/api/quoteCartQuerySelectors.ts:90]: I am sure you are right, I was just going by the data type that I have in the sidebar... Maybe I should make this function take a QuoteBundleVariant and then try to get the QuoteBundleVariant in the sidebar?

**Review by dendoesit — APPROVED** [review]
good work 🎉

File: src/client/api/quoteCartQuerySelectors.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (amandaflod) [reply src/client/api/quoteCartQuerySelectors.ts:90]: I will leave it as it is, unfortunately it is a bit messy with the different data types but not much to do 😅

**Pull request merged**